sites/public/pages/index.tsx

```diff
@@ -58,7 +58,7 @@ export default function Home(props: IndexProps) {
         title={heroTitle}
         buttonTitle={t("welcome.seeRentalListings")}
         buttonLink="/listings?page=1"
-        listings={props.listings}
+        children={t("welcome.subTitle")}
```

Comment:
I think children are typically done like an inner tag instead of passing something in as a child prop directly. See `_error.jsx` as an example.

Reply:
Updated